modules/client/test/unit/client-common/utils.test.ts

```diff
@@ -76,3 +121,40 @@ describe("client-common utils", () => {
     });
   });
 });
+
+describe("Compute failing proposals", () => {
+  it("Should return true because the proposal changes the min approvals and then updates the addresses", async () => {
+    const actions: DaoAction[] = [
+      UPDATE_MULTISIG_SETTINGS_ACTION,
+      ADD_ADDRESSES_ACTION,
+    ];
+    expect(isFailingProposal(actions)).toBe(
+      true,
+    );
+  });
+  it("Should return true because the proposal removes addresses and then updates the settings", async () => {
+    const actions: DaoAction[] = [
+      REMOVE_ADDRESSES_ACTION,
+      UPDATE_MULTISIG_SETTINGS_ACTION,
+    ];
+    expect(isFailingProposal(actions)).toBe(
+      true,
+    );
+  });
+  it("Should return false because the proposal adds addresses, changes the min approvals and then removes addresses", async () => {
```

Comment:
```suggestion
  it("isFailingProposal should return false because the proposal adds addresses, changes the min approvals and then removes addresses", async () => {
```